### ISSUE TITLE
Drag and drop does not work in reaper.

### ISSUE THREAD

**Issue by TheGuyWhoo** [rorywalsh/cabbage#133]
linux mint 20, reaper 6.44, cinnamon, cabbage 2.8.0.

[dndtest.csd.zip](https://github.com/rorywalsh/cabbage/files/9274833/dndtest.csd.zip)

this works in cabbage, but not in reaper.

**Comment by rorywalsh** [comment]
Yes, I get the same behaviour here. Leave it with me. Hopefully it's a simple fix.

**Comment by rorywalsh** [comment]
I just took a look now. It works for me when Reaper when the UI is integrated into the plugin window in Reaper. But when I have to hit the _Show UI_ button the dropped file is not picked up. It looks like Reaper's bridged plugin UI is not passing on the dropped file event to the plugin UI. If this is the case I'm not sure I can do much about it. 

Btw, you should use one of the later versions if you can. 2.8.0 is pretty old at this stage. Are you building from source? If not you can grab the latest Linux builds from [here](https://dev.azure.com/rorywalsh/cabbage/_build?view=runs). There was an issue with the Linux CI build but I think it's fixed now. You should be able to access it when it's done building.

**Comment by TheGuyWhoo** [comment]
So it works for you if the plugin window is not floating correct? Or do you mean you have "run as native only' selected? 

Just tried that and it's not working for me. I will update tomorrow and let you know if that fixes the issue.

**Comment by rorywalsh** [comment]
Yes, it does not work for when the plugin appears in the floating window. But I don't have access to a Linux machine right now as I'm away from my studio. But on MacOS it works Ok.

**Comment by TheGuyWhoo** [comment]
Hey so I'm finally getting around to this, but as far I can tell there is no artifacts to be found in the link you provided.
